tests/admin_inlines/models.py

```diff
@@ -174,6 +175,25 @@ class FootNote(models.Model):
     chapter = models.ForeignKey(Chapter, models.PROTECT)
     note = models.CharField(max_length=40)
 
+
+# Models for #15665
```

Comment:
Could you try to reuse existing models? We try to minimize the number of models because they increase the time it takes to run the tests. It looks like adding this should work:
```
class LifeFormAdmin(admin.ModelAdmin):
    inlines = [SightingInline]
```